pkg/kubelet/prober/prober.go

```diff
@@ -145,7 +145,9 @@ func (pb *prober) runProbe(p *api.Probe, pod *api.Pod, status api.PodStatus, con
 		path := p.HTTPGet.Path
 		glog.V(4).Infof("HTTP-Probe Host: %v://%v, Port: %v, Path: %v", scheme, host, port, path)
 		url := formatURL(scheme, host, port, path)
-		return pb.http.Probe(url, timeout)
+		headers := p.HTTPGet.HTTPHeaders
+		glog.V(1).Infof("HTTP-Probe Headers: %v", headers)
```

Comment:
This is too chatty - it will log every probe of every container.  V(4) seems correct